src/ONYXKEYS.ts

```diff
@@ -449,10 +449,10 @@ const ONYXKEYS = {
          */
         WORKSPACE_CARDS_LIST: 'card_',
 
-        /** The bank account that Expensify Card payments will be reconciled against */
+        /** Stores which connection is set up to use Continuous Reconciliation */
         SHARED_NVP_EXPENSIFY_CARD_CONTINUOUS_RECONCILIATION_CONNECTION: 'sharedNVP_expensifyCard_continuousReconciliationConnection_',
```

Comment:
@MariaHCD @mountiny - just to make sure, sharedNVP_expensifyCard_continuousReconciliationConnection_ should store connection name, not the bank account data?

Reply:
Yes, sharedNVP_expensifyCard_continuousReconciliationConnection_ will just store the name of the connection like `quickbooksOnline`, `xero`, `netsuite` or `intacct`

Reply:
ok, thanks! So it's now fixed in this PR

Reply:
It will just be a string. I believe it's only one connection at a time.